kommunicate/src/main/java/io/kommunicate/utils/KmUtils.java

```diff
@@ -77,6 +78,11 @@ public static void setDrawableTint(TextView textView, int colorId, int index) {
                 .setColorFilter(new PorterDuffColorFilter(colorId, PorterDuff.Mode.SRC_IN));
     }
 
+    public static void setDrawableTint(ImageView imageView, int colorId) {
```

Comment:
from where we are calling this function?

Reply:
Earlier we were calling `setDrawableTint(Textview ...)` from agent app, but now its not called from anywhere.
We are now calling `setDrawableTint(ImageView ...)` from agent app. 

![image](https://github.com/Kommunicate-io/Kommunicate-Android-Chat-SDK/assets/162960364/26491799-b980-448e-a34b-571b6fdc0680)

Same PR is raised in agent app too, since there is modification in both. https://github.com/Kommunicate-io/AgentApp/pull/321/files